Chandra/cmd_states/tests/test_get_cmd_states.py

```diff
@@ -45,7 +46,7 @@ def test_get_states_main():
 
 # Set up possible backends
 dbis = ['hdf5', 'sqlite']
-if six.PY2:
+if six.PY2 and os.path.exists(os.environ['SYBASE']):
```

Comment:
I thought that was a guarantee in any SKA environment, but that would be a good extra check in this line.